content/200-orm/200-prisma-client/150-using-raw-sql/100-typedsql.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'TypedSQL'
+metaTitle: 'Writing Type-safe SQL with TypedSQL and Prisma Client'
+metaDescription: 'Learn how to use TypedSQL to write fully type-safe SQL queries that are compatible with any SQL console and Prisma Client.'
+---
+
+## Getting started with TypedSQL
+
+To start using TypedSQL in your Prisma project, follow these steps:
+
+1. Ensure you have `@prisma/client` and `prisma` installed and updated to at least version `5.19.0`.
+
+2. Create a `sql` directory inside your `prisma` directory. This is where you'll write your SQL queries.
+
+   ```terminal
+   cd prisma
+   mkdir sql
+   ```
+
+3. Write your SQL queries in `.sql` files within the `sql` directory. For example:
```

Comment:
Couple fo requirements to file name:

1. It has to be a valid JS identifier
2. It can not start with `$` (reserved for internal usage by us)